tensorboard/plugins/graph/tf_graph_common/BUILD

```diff
@@ -43,3 +44,18 @@ tf_ts_library(
         "@npm//lodash",
     ],
 )
+
+tf_ts_library(
+    name = "test_lib",
+    testonly = True,
+    srcs = [
+        "common_test.ts",
+    ],
+    deps = [
+        ":tf_graph_common",
+        "//tensorboard/webapp/tb_debug",
+        "@npm//@types/jasmine",
+        "@npm//dagre",
+        "@npm//lodash",
```

Comment:
Do you need dagre and lodash here? It seems to violate strict_deps :\

Reply:
Done!